core/src/com/unciv/ui/audio/MusicMood.kt

```diff
@@ -7,6 +7,9 @@ object MusicMood {
     const val Defeat = "Defeat"
     const val Menu = "Menu"
     const val Ambient = "Ambient"
+    const val Golden = "Golden"
+    const val Built = "Built"
```

Comment:
if this is the `Wonder built` mood I'd think `Wonder` or `WonderBuilt` would be clearer

Reply:
Good point.